### ISSUE TITLE
Detecting videos in M3U8 format is not supported

### ISSUE THREAD

**Issue by maxfrees** [amir1376/ab-download-manager#95]
Detecting videos in M3U8 format is not supported

**Comment by amir1376** [comment]
@maxfrees is this related to #9 ?

**Comment by maxfrees** [comment]
@amir1376 Yes, the features look similar, but not just in mp4 , but also in m3u8 format

**Comment by amir1376** [comment]
@maxfrees Ok so. I will look into this when I'm going to add download stream feature. Thank you.🙏

**Comment by maxfrees** [comment]
> @maxfrees Ok so. I will look into this when I'm going to add download stream feature. Thank you.🙏

OK, this feature is very important and I hope it will replace IDM

**Comment by plasx520** [comment]
是的M3U8这个功能非常重要
